src/main/resources/feedbackQuestionConstSumEditFormOptionFragment.html

```diff
@@ -1,5 +1,5 @@
-<div id="constSumOptionRow-${i}-${questionNumber}">
-    <div class="input-group col-sm-12">
+<div class="margin-bottom-7px" id="constSumOptionRow-${i}-${questionNumber}">
+    <div class="input-group width-100-pc">
```

Comment:
Yeah, I did! :-P
Since this is an atomic class, the convention is to name it "attribute-value". Here its width: 100%. Any other way to name this?